pkg/apiclient/auth.go

```diff
@@ -44,6 +44,10 @@ func (t *APIKeyTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 	// Make the HTTP request.
 	resp, err := t.transport().RoundTrip(req)
+	if resp == nil {
```

Comment:
idiomatic way of doing this is testing err being no-nil. Are you doing something different, here ?

Reply:
i didn't wrote this code. But my code was crashing because `resp` was equal to `nil`.